include/z64audio.h

```diff
@@ -1068,13 +1068,13 @@ typedef struct {
 } unk_D_8016E750; // size = 0x264
 
 typedef enum {
-    BANK_PLAYER,
-    BANK_ITEM,
-    BANK_ENV,
-    BANK_ENEMY,
-    BANK_SYSTEM,
-    BANK_OCARINA,
-    BANK_VOICE
+    BANK_PLAYER = 0,
+    BANK_ITEM = 1,
+    BANK_ENV = 2,
+    BANK_ENEMY = 3,
+    BANK_SYSTEM = 4,
+    BANK_OCARINA = 5,
+    BANK_VOICE = 6
```

Comment:
We usually just comment the index for these.

Reply:
Specifically, on the left. You can see plenty of examples in the codebase.